be/src/exec/pipeline/result_sink_operator.h

```diff
@@ -63,7 +63,7 @@ class ResultSinkOperator final : public Operator {
     std::atomic<int64_t>& _num_written_rows;
 
     std::shared_ptr<ResultWriter> _writer;
-    mutable TFetchDataResultPtr _fetch_data_result;
+    mutable TFetchDataResultPtrs _fetch_data_result;
```

Comment:
Why change it to a vector?

Reply:
> Why change it to a vector?

before my change,  `MysqlResultWriter::process_chunk` will return only one TFetchDataResultPtr, but now It may return multiple